### ISSUE TITLE
Performance of VGG16 based Deeplab-v1

### ISSUE THREAD

**Issue by PengtaoJiang** [halbielee/EPS#12]
Hello, @halbielee , 
    I use the pseudo labels to train the VGG16 based Deeplab-v1 used in OAA.  However, I only get 65.1% mIOU.  Could you please tell me where is the difference between my model and yours?

**Comment by halbielee** [comment]
Hello @PengtaoJiang ,

As you know, there can be many factors for the difference.

For the VGG16 based DeepLab-V1, we do not use the DeepLab-V1 used in OAA. We reproduce it from [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch).

Also, the pseudo-mask can be different from you used.

Here is the pseudo-masks we used for the paper. [Link for the pseudo-masks](https://drive.google.com/file/d/1JUux3TsejtWz5n__cxAB0-vwKMPYCA_4/view?usp=sharing).

If you have more question, please do not hesitate. Thanks!

**Comment by PengtaoJiang** [comment]
Thanks!
